sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_receiver.py

```diff
@@ -985,3 +1004,15 @@ def renew_message_lock(
         )
 
         return message._expiry  # type: ignore
+
+    @property
+    def client_identifier(self) -> str:
+        """
+        Get the ServiceBusReceiver client_identifier associated with the receiver instance.
+
+        :rtype: str
+        """
+        return self._name
+
+    def __str__(self) -> str:
+        return f"ServicebusReceiver client identifier: {self.client_identifier} for entity: {self.entity_path}"
```

Comment:
Nitpick; I'd make this shorter. The pertinent information can get hidden when the string is this long (think what you would like to see in a log or in a debugger/repl window)

Reply:
awesome, made it a little shorter :)